packages/assert/src/index.ts

```diff
@@ -59,6 +59,7 @@ export const assert = {
     isBoolean(variableName: string, value: boolean): void {
         assert.assert(_.isBoolean(value), assert.typeAssertionMessage(variableName, 'boolean', value));
     },
+    // DEPRECATED: Please use providerUtils.standardizeOrThrow() instead
```

Comment:
This comment will live here forever and no one will ever see it. Can we use `logUtils.warn` instead?